.travis/debian.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -xe
+
+# before_install
+docker pull debian:stable
+docker run --name travis-ci -v $TRAVIS_BUILD_DIR:/primitiv-python -td debian:stable /bin/bash
+
+# install
+docker exec travis-ci bash -c "apt update"
+docker exec travis-ci bash -c "apt install -y git build-essential cmake python3-dev python3-pip python3-numpy"
+docker exec travis-ci bash -c "pip3 install cython scikit-build"
+
+# install OpenCL environment
+docker exec travis-ci bash -c "apt install -y opencl-headers libclblas-dev pkg-config libhwloc-dev libltdl-dev ocl-icd-dev ocl-icd-opencl-dev clang-3.8 llvm-3.8-dev libclang-3.8-dev libz-dev"
+# pocl 0.13 does not contain mem_fence() function that is used by primitiv.
+# We build the latest pocl instead of using distribution's package.
+# See: https://github.com/pocl/pocl/issues/294
+docker exec travis-ci bash -c "git clone https://github.com/pocl/pocl.git"
+docker exec travis-ci bash -c "cd ./pocl && cmake . -DCMAKE_INSTALL_PREFIX=/usr"
+docker exec travis-ci bash -c "cd ./pocl && make && make install"
+
+if [ "${WITH_CORE_LIBRARY}" = "yes" ]; then
+    # script
+    docker exec travis-ci bash -c "cd /primitiv-python && git clone https://github.com/primitiv/primitiv.git primitiv-core"
```

Comment:
Should specify a commit number, or include the core library as a Git submodule. The HEAD branch of the core library does not guarantee to pass any external tests.

Reply:
`primitiv-core` is added as a sub module in `feature/setup-bundle-core`.
I will fix this branch after `feature/setup-bundle-core` is merged.